doc/stdenv/stdenv.chapter.md

```diff
@@ -116,18 +116,19 @@ On Linux, `stdenv` also includes the `patchelf` utility.
 
 ## Specifying dependencies {#ssec-stdenv-dependencies}
 
-As described in the Nix manual, almost any `*.drv` store path in a derivation’s attribute set will induce a dependency on that derivation. `mkDerivation`, however, takes a few attributes intended to, between them, include all the dependencies of a package. This is done both for structure and consistency, but also so that certain other setup can take place. For example, certain dependencies need their bin directories added to the `PATH`. That is built-in, but other setup is done via a pluggable mechanism that works in conjunction with these dependency attributes. See <xref linkend="ssec-setup-hooks" /> for details.
+While _derivations_ collect their dependencies by references to store paths of other _derivations_, `mkDerivation`, furthermore deals with _package_ depdencies along four axes: build, host & target platform and wether they are propagated. Details can be found in <xref linkend="chap-cross" />, but, for quick reminder, the machine you are building on is _build_, the machine that you are building for _host_, and the machine that the compiler will produce code for _target_. If build, host, and target are all the same, this is called a _native_. If build and host are the same but target is different, this is called a _cross_. If host and target are the same, but build is different, you are using a _cross-compiler_ to build a _native_ for a different system, a _cross-built native_, because target is the machine that the compiler will produce code for, while running on build.
```

Comment:
```suggestion
While _derivations_ collect their dependencies by references to store paths of other _derivations_, `mkDerivation`, furthermore deals with _package_ depdencies along four axes: build, host & target platform and wether they are propagated. Details can be found in <xref linkend="chap-cross" />, but, for quick reminder, 
- the machine you are building on is _build_, 
- the machine that you are building for _host_, 
- and the machine that the compiler will produce code for _target_. 

Accordingly:
- If build, host, and target are all the same, this is called a _native_. 
- If build and host are the same but target is different, this is called a _cross_.
- If host and target are the same, but build is different, you are using a _cross-compiler_ to build a _native_ for a different system, a _cross-built native_, because target is the machine that the compiler will produce code for, while running on build.
```